contracts/treasury/LiquidationStrategyBase.sol

```diff
@@ -1,18 +1,20 @@
 // SPDX-License-Identifier: agpl-3.0
 pragma solidity 0.7.6;
 
-import {PermissionAdmin} from '@kyber.network/utils-sc/contracts/PermissionAdmin.sol';
+import {PermissionAdmin, PermissionOperators} from '@kyber.network/utils-sc/contracts/PermissionOperators.sol';
 import {Utils} from '@kyber.network/utils-sc/contracts/Utils.sol';
 import {IERC20Ext} from '@kyber.network/utils-sc/contracts/IERC20Ext.sol';
 import {ReentrancyGuard} from '@openzeppelin/contracts/utils/ReentrancyGuard.sol';
 import {SafeMath} from '@openzeppelin/contracts/math/SafeMath.sol';
 import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
 import {EnumerableSet} from '@openzeppelin/contracts/utils/EnumerableSet.sol';
-import {ILiquidationStrategy} from '../interfaces/liquidation/ILiquidationStrategy.sol';
 import {ILiquidationCallback} from '../interfaces/liquidation/ILiquidationCallback.sol';
+import {ILiquidationStrategyBase} from '../interfaces/liquidation/ILiquidationStrategyBase.sol';
+import {ILiquidationPriceOracleBase} from '../interfaces/liquidation/ILiquidationPriceOracleBase.sol';
```

Comment:
can change to `import {ILiquidationPriceOracleBase, ILiquidationStrategyBase} from '../interfaces/liquidation/ILiquidationStrategyBase.sol';`, then delete line above

Reply:
Updated